<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeMapsCentral</class>
 <widget class="QWidget" name="ModeMapsCentral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="CollectionDisplay" name="collDisp" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>10</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="ThemaMapView" name="mapView" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="PObjectIconView" name="contentsList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ThemaMapView</class>
   <extends>QWidget</extends>
   <header>gui/mapviews/themamapview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CollectionDisplay</class>
   <extends>QWidget</extends>
   <header>gui/forms/collectiondisplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PObjectIconView</class>
   <extends>QListView</extends>
   <header>gui/forms/pobjecticonview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
